srcs/http/response/http_method.cpp

```diff
@@ -204,4 +222,40 @@ bool Method::IsAllowedMethod(
 	}
 }
 
+// ./と../はいらないかも？
+utils::Result<void>
+Method::AutoindexHandler(const std::string &path, std::string &response_body_message) {
+	utils::Result<void> result;
+	DIR                *dir = opendir(path.c_str());
+
+	if (dir == NULL) { // system exception?
+		result.Set(false);
+		return result;
+	}
+
+	struct dirent *entry;
+	response_body_message += "<html>\n"
+							 "<head><title>Index of /</title></head>\n"
+							 "<body><h1>Index of /</h1><hr><pre>"
+							 "<a href=\"../\">../</a>\n";
+	while ((entry = readdir(dir)) != NULL) {
+		std::string full_path = std::string(path) + "/" + entry->d_name;
```

Comment:
`path` 元々 string なので変換しなくて大丈夫そうです
```suggestion
		std::string full_path = path + "/" + entry->d_name;
```